apps/prax-marketing-site/vite.config.ts

```diff
@@ -2,6 +2,7 @@ import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react-swc';
 
 export default defineConfig({
+  clearScreen: false,
```

Comment:
what does this do?

Reply:
this prevents the vite build from blanking the terminal, so extension build output remains visible when running `dev` from the repository root